docs/tasks-yaml-format.md

```diff
@@ -8,7 +8,7 @@ There are five top level fields in the spec:
 category: sample
 difficulty: Easy
 objective: Sample yaml
-kind: cp.simulator/scenario:<semver-version>
+kind: cp.simulator/scenario:<server-version>
```

Comment:
`semver` is correct I think. It's referring to semantic versioning for the tag

Reply:
semantic version - version?